_docs/migrate_module.md

```diff
@@ -10,12 +10,12 @@ You will have someone by your side in this process. The general flow is to…
 * Ask one of the Administrators to add you to the `incoming-migrations` team.
 * At that point you can transfer your own repository.
 * If migrating a module from puppetlabs, re-enable github issues.
-* Verify that all webhooks except travis are disabled.
+* Verify that all webhooks are disabled.
 * Update the README.md with a description of the deprecation and a link to the new module location.
 * Release a copy of your module to the 'puppet' forge account.
-* Add the module to our [modulesync setup](managed_modules).
-* Add the module to our [plumbing repository](plumbing)(handles travis secrets).
-* Ask an admin to add the `collaborators` team to the module's `Collaborators & Teams` 'Teams' list with `Write` permissions (e.g. [https://github.com/voxpupuli/puppet-gitlab/settings/collaboration](https://github.com/voxpupuli/puppet-gitlab/settings/collaboration))).
+* Add the module to our [modulesync setup](publish_setup).
+* Ask an admin to add the `collaborators` team to the module's `Collaborators & Teams` 'Teams' list with `Write` permissions (e.g. [https://github.com/voxpupuli/puppet-gitlab/settings/collaboration](https://github.com/voxpupuli/puppet-gitlab/settings/collaboration)))
```

Comment:
Sorry, bad suggestion. This keeps the diff smaller (I think)
```suggestion
* Ask an admin to add the `collaborators` team to the module's `Collaborators & Teams` 'Teams' list with `Write` permissions (e.g. [https://github.com/voxpupuli/puppet-gitlab/settings/collaboration](https://github.com/voxpupuli/puppet-gitlab/settings/collaboration))).
```

Reply:
I think there should only be two `)` in the end and not three?